com.gitlab.fossbr.WhatsAppQT.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally, you should maintain it in your GitLab repo.

Reply:
Additionally, it doesn't look like you are installing this file at all.

Reply:
additionaly is looks like now there are to appdata file in this repository.